### PR TITLE
Announce discuss.lagomframework.com

### PR DESCRIPTION
**Pull request by TimMoore**
- Add a blog post with an introduction
- Change all mailing list links to the new forum
- Fix small typos and other issues on the affected pages

### CODE REVIEW

**Comment by ignasi35** [comment]
Wow! That's a hell of an improvement. A part from the announce the whole review looks awesome. Good job!

File: src/main/markdown/faq.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 Have a question about Lagom? We've sorted out a sample of frequently asked questions and provided answers below. If your question is not covered here, try these resources:
 
+- [Discuss Lagom forum](https://discuss.lagomframework.com)
```
Comment (renatocaval) [inline src/main/markdown/faq.md]: good to have it as the first and preferred channel. Gitter second. :-)

File: src/main/assets/_social.styl [hunk 2]
```diff
@@ -89,7 +89,7 @@
 			social(3)
 		&[href*=stackoverflow]
 			social(4)
-		&[href*=forum]
+		&[href*=discuss]
```
Comment (TimMoore) [inline src/main/assets/_social.styl]: This had me scratching my head for a bit... strange but I didn't want to redo it now.

Comment (renatocaval) [reply src/main/assets/_social.styl]: that's is that actually?

**Comment by TimMoore** [comment]
One unfortunate issue is that we're still using an envelope icon for the forums, because we don't have anything else in the project that makes more sense and matches the other ones. Something to fix in a future iteration of the site.

File: src/main/assets/_social.styl (hunk 2 above)
Comment (renatocaval) [reply src/main/assets/_social.styl]: hmm, I guess these are the icons on on the left-upper corner. Maybe we should remove the mailing list one.

Comment (TimMoore) [reply src/main/assets/_social.styl]: It's applying a style based on a substring match of the URL!

The style is used to set the background image offset to display the correct icon from a sprite image.

Comment (TimMoore) [reply src/main/assets/_social.styl]: I think it's good to have a link to the forum there... the icon is unfortunate but we can fix later.

**Pull request merged**